### PR TITLE
updated plausible script to include pageview

### LINKED ISSUE #234: Add New Stats to Plausible
We need to track which school pages are being visited. Add page tracking for all the school pages (including their query strings)

### PR DESCRIPTION
**Pull request by BeeSeeWhy**
closes #234

### CODE REVIEW

**Comment by BeeSeeWhy** [comment]
I don't know if I will be able to tell if this works without merging

**Comment by mattgianni** [comment]
Maybe Docker could help with the testing by running plausible locally in a container.

**Comment by nickvisut** [comment]
Welp, there's one way to find out if there's no time for Docker atm

File: src/pages/_document.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ class MyDocument extends Document {
           <script
             defer
             data-domain="supportsfschools.org"
-            src="https://plausible.io/js/script.tagged-events.outbound-links.js"
+            src="https://plausible.io/js/script.pageview-props.tagged-events.outbound-links.js"
```
Comment (nickvisut) [inline src/pages/_document.tsx:65]: lgtm according to docs

**Review by nickvisut — APPROVED** [review]

**Pull request merged**